### ISSUE TITLE
How to use the base model weight?

### ISSUE THREAD

**Issue by qiqiApink** [PRIS-CV/Omnieraser#3]
The `in_channels` in the transformer has been modified. However, only lora weights have been released.

**Comment by RunpuWei** [comment]
Thanks for your interest, we have made our HuggingFace demo public,  you can use our demo code to load weights.

**Comment by qiqiApink** [comment]
The demo link seems wrong. Btw, I implemented the inference code but got a bad result. Can u give me some advice?

```
flux_transformer = FluxTransformer2DModel.from_pretrained(
    'FLUX.1-dev',
    subfolder="transformer",
)
with torch.no_grad():
    initial_input_channels = flux_transformer.config.in_channels
    new_linear = torch.nn.Linear(
        flux_transformer.x_embedder.in_features*4,
        flux_transformer.x_embedder.out_features,
        bias=flux_transformer.x_embedder.bias is not None,
        dtype=flux_transformer.dtype,
        device=flux_transformer.device,
    )
    new_linear.weight.zero_()
    new_linear.weight[:, :initial_input_channels].copy_(flux_transformer.x_embedder.weight)
    if flux_transformer.x_embedder.bias is not None:
        new_linear.bias.copy_(flux_transformer.x_embedder.bias)
    flux_transformer.x_embedder = new_linear
flux_transformer.register_to_config(in_channels=initial_input_channels*4, out_channels=initial_input_channels)
pipeline = FluxControlRemovalPipeline.from_pretrained(
    'FLUX.1-dev',
    transformer=flux_transformer.to(torch.bfloat16),
    torch_dtype=torch.bfloat16,
)
pipeline.load_lora_weights('pytorch_lora_weights.safetensors')
pipeline = pipeline.to('cuda')

validation_image = load_image(validation_image_path)
validation_image = validation_image.resize((1024, 1024))
validation_mask = load_image(validation_mask_path)
validation_mask = validation_mask.resize((1024, 1024))
validation_prompt = 'There is nothing here.'

with torch.no_grad():
    image = pipeline(
        prompt=validation_prompt,
        control_image=validation_image,
        control_mask=validation_mask,
        max_sequence_length=512,
        num_inference_steps=20,
        height=1024,
        width=1024,
        guidance_scale=3.5,
        generator=torch.Generator('cuda').manual_seed(42),
    ).images[0]
```

**Comment by RunpuWei** [comment]
Sorry, I have fixed the demo link error, now you can access the online demo normally. I am still actively sorting out the code. There may be some bugs, I'll let you know if the problems are solved in the future.

**Comment by qiqiApink** [comment]
Does the demo not support uploading a mask? I have got `ZeroGPU worker error` when testing with my custom image and mask.

**Comment by RunpuWei** [comment]
I think it may be the implementation problem of ZeroGPU. 

The same as ClipAway (https://huggingface.co/spaces/yigitekin/CLIPAway), we encountered the problem that the example could not be imported if the drawing board was implemented in Huggingface Space.

 I'm still testing the code extensively and trying to fix this compatibility BUG later.

And can you give me some of your bad cases, I will help you analyze.

**Comment by Sure66666** [comment]
> Does the demo not support uploading a mask? I have got `ZeroGPU worker error` when testing with my custom image and mask.

We have fixed the code error, and now you can try it again~

**Comment by qiqiApink** [comment]
Thanks a lot!!!

**Comment by tenghui98** [comment]
still got ZeroGPU worker error when uploading image

**Comment by Sure66666** [comment]
> still got ZeroGPU worker error when uploading image
We have fixed the code error, and now you can try it again~.

**Comment by RunpuWei** [comment]
News, we have release Omnieraser Model weights (ControlNet version), which enables more accurate object removal with significantly better background consistency, thanks to the assistance of alimama ControlNet. For this weight we consider only the purest object removal, you can find the code and the gradio demo in the ControlNet_version folder of this repository!!
